src/commcare_cloud/ansible/deploy_commcarehq.yml

```diff
@@ -73,22 +73,22 @@
   tags: services
 
 - name: Management Command Supervisor Config
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
```

Comment:
If the HQ code isn't on formplayer machines then we can't run management command from them either.

```suggestion
  hosts: commcarehq
```

Reply:
What management commands do we need to run on a formplayer machine?

Reply:
@millerdev none, hence why I suggested removing it.

Reply:
I'm sorry, I misinterpreted your comment. I thought you were saying we need to keep the commcarehq source on these machines so we can run management commands on them.